lib/session_management.h

```diff
@@ -13,16 +13,13 @@
 
 #include <config.h>
 
+#include <stdlib.h>
 
-/**
- * @brief Get host for the current session
- *
- * @param[out] out Host name
- *
- * @return 0 or a positive integer if the host was obtained properly,
- *         another value on error.
- */
```

Comment:
You can update this comment and keep it for future users of the API

Reply:
I prefer not.  None of the documentation I see in the existing comment are not present in the prototype.

```c
$ grepc -htfp get_session_host .
ATTR_MALLOC(free)
char *get_session_host(void);
```

From the name, we know we get the "host".  From `char *` we know it's a string.  From `ATTR_MALLOC(free)` we know the called needs to free(3) the string.

And the name more or less says "get host for the current session".